src/components/reviews.tsx

```diff
@@ -26,32 +26,24 @@ const useStyles = makeStyles((theme: Theme) =>
 
 export const Reviews: React.FC<{ id: string }> = (props) => {
   const classes = useStyles();
-  const [reviews, setRewiews] = React.useState<Review[]>([]);
+  const [reviews, setReviews] = React.useState<Review[]>();
   const { id } = props;
-
-  if (!id) {
-    return null;
-  }
-
-  console.log(reviews, id);
-
-  React.useEffect(() => {
+  if(!reviews){
     fetch(EndPoint.reviews(id))
       .then((response) =>
         response.url.endsWith(".yaml")
           ? response.text().then((text) => yaml.safeLoad(text))
           : response.json()
       )
-      .then((data) => setRewiews(data))
+      .then((data) => setReviews(data))
       .catch((error) => {
         console.log("unable to fetch data", error);
       });
-  }, [id]);
-
+      return <></>
+  }
   if (!Array.isArray(reviews)) {
-    return null;
+      return <></>;
```

Comment:
lets's keep it null.
More in this here - https://stackoverflow.com/questions/42083181/is-it-possible-to-return-empty-in-react-render-function